turbinia/evidence.py

```diff
@@ -77,6 +78,8 @@ class Evidence(object):
         processing this evidence.
     cloud_only: Set to True for evidence types that can only be processed in a
         cloud environment, e.g. GoogleCloudDisk.
+    context_dependant: Whethere this evidence required to be built upon the
+        context of a parent evidence.
```

Comment:
Don't we need to set some of the other evidence types as context dependent then?  Or does that get done in the next PR when we actually move around the pre/post-processors?

Reply:
There is currently no other evidence type where this is absolutely needed, but a later PR could indeed make GoogleCloudDiskRawEmbedded dependent to GoogleCloudDisk